.azure_pipelines/ci-pipeline-code-coverage-nightly.yml

```diff
@@ -37,11 +36,11 @@ jobs:
 
     # build all source files
     - script: |
-        make clean && make
+        sudo rm -rf `git ls-files --others --directory` && make -j
```

Comment:
We probably still need "make distclean" besides "sudo rm -rf `git ls-files --others --directory`"

Reply:
Hi, Yes, it removes anything that is not tracked in deislabs/mystikos

Reply:
I added it back in the yaml for consistency though